source/examples/vulkan/vk_color_cube/vk_color_cube.cc

```diff
@@ -421,7 +421,6 @@ bool AMDVulkanDemo::InitializeGpa()
     if (kGpaStatusOk != status)
     {
         AMDVulkanDemoVkUtils::Log("ERROR: GpaGetFuncTable failed with status %d", status);
-        delete gpa_function_table;
         return false;
     }
```

Comment:
Are you sure this gpa_function_table is supposed to be deleted? It gets allocated earlier in this function, so it should probably be deleted here to avoid the small memory leak.